server/graphql/core/src/test_helpers.rs

```diff
@@ -106,10 +106,9 @@ async fn graphql<Q: 'static + ObjectType + Clone, M: 'static + ObjectType + Clon
     schema.execute(query).await.into()
 }
 
-// TODO should really re-export dev deps (actix_rt, assert_json_dif, to avoid need to import in consumer)
 #[macro_export]
-macro_rules! assert_graphql_query {
-    ($settings:expr, $query:expr, $variables:expr, $expected_inner:expr, $service_provider_override:expr) => {{
```

Comment:
need to compare array exactly (for breach_ids in temperature chart), otherwise lhs ["1"] and rhs ["1", "2"] was asserting as true